### DISCUSSION TITLE
Problems running A1111 on Google Collab

### DISCUSSION THREAD

**Discussion by williamsblood** [TheLastBen/fast-stable-diffusion#2616] [Q&A]
Even after Google banned SD from Colab, I was able to use it until yesterday (Via Colab Pro) but today when I tried to run the notebook I got this error:

WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
     PyTorch 2.0.1+cu118 with CUDA 1108 (you have 2.1.0+cu118)
     Python 3.10.11 (you have 3.10.12)
   Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
   Memory-efficient attention, SwiGLU, sparse and more won't be available.
   Set XFORMERS_MORE_DETAILS=1 for more details

Could anyone help?

**Comment by kiskejp** [comment]
I got the same error.
Should I downgrade Python? But I don't know how to do that...

**Comment by Marsh-6** [comment]
Getting the same error since today. Did some research and seems like the solution is simple, but I just don't have enough code knowledge to implement it. Tried a clean reinstall but got the same error.

If someone could let us know how to implement this solution: https://github.com/facebookresearch/xformers/issues/664

Thank you! :)

**Reply by ClaraSanders** [reply]
Welcome to the amazing world of Google! I'm getting used to the Colab life. In fact, I can't believe A1111 worked without any issues for more than a month. We'll have to wait for Ben to fix it like always. I hope his doggo (from his avatar) is fine.

**Reply by williamsblood** [reply]
A working around here: https://github.com/TheLastBen/fast-stable-diffusion/issues/2615#issuecomment-1771708452

**Reply by kiskejp** [reply]
worked for me! thanks!

**Reply by Marsh-6** [reply]
Worked yesterday but stopped working again today :(

**Reply by TheLastBen** [reply]
> I hope his doggo (from his avatar) is fine.

She left this world a few years ago, sad world after that.

**Reply by ClaraSanders** [reply]
I'm really sorry. I know very well pets are irreplaceable but I hope another one fins its place in your heart.

**Comment by TheLastBen** [comment]
Use the latest notebook

**Reply by toeythewolf04** [reply]
Thank you for your updated sir. It's better but still error occur.

**Reply by toeythewolf04** [reply]
Tried to run and it's mostly done. But in the end of process. error occur after proceed. This was my error message >>

NotImplementedError: No operator found for `memory_efficient_attention_forward` with inputs: query : shape=(1, 5525, 1, 512) (torch.float32) key : shape=(1, 5525, 1, 512) (torch.float32) value : shape=(1, 5525, 1, 512) (torch.float32) attn_bias : <class 'NoneType'> p : 0.0 `decoderF` is not supported because: max(query.shape[-1] != value.shape[-1]) > 128 xFormers wasn't build with CUDA support attn_bias type is <class 'NoneType'> operator wasn't built - see `python -m xformers.info` for more info `flshattF@0.0.0` is not supported because: max(query.shape[-1] != value.shape[-1]) > 256 xFormers wasn't build with CUDA support requires device with capability > (8, 0) but your GPU has capability (7, 5) (too old) dtype=torch.float32 (supported: {torch.bfloat16, torch.float16}) operator wasn't built - see `python -m xformers.info` for more info `tritonflashattF` is not supported because: max(query.shape[-1] != value.shape[-1]) > 128 xFormers wasn't build with CUDA support requires device with capability > (8, 0) but your GPU has capability (7, 5) (too old) dtype=torch.float32 (supported: {torch.bfloat16, torch.float16}) operator wasn't built - see `python -m xformers.info` for more info triton is not available requires GPU with sm80 minimum compute capacity, e.g., A100/H100/L4 `cutlassF` is not supported because: xFormers wasn't build with CUDA support operator wasn't built - see `python -m xformers.info` for more info `smallkF` is not supported because: max(query.shape[-1] != value.shape[-1]) > 32 xFormers wasn't build with CUDA support operator wasn't built - see `python -m xformers.info` for more info unsupported embed per head: 512

**Reply by toeythewolf04** [reply]
OH, I saw the updated.

no module 'xformers'. Processing without...

I can run it now.
Thank you very much sir.

**Reply by TheLastBen** [reply]
Use the latest notebook, all is fixed now
